george.planx-pla.net/manifest.json

```diff
@@ -0,0 +1,173 @@
+{
+  "notes": [
+    "This is the dev environment manifest",
+    "That's all I have to say"
+  ],
+  "jenkins": {
+    "autodeploy": "yes"
+  },
+  "versions": {
+    "arborist": "quay.io/cdis/arborist:master",
+    "arranger": "quay.io/cdis/arranger:master",
+    "arranger-adminapi": "quay.io/cdis/arranger-server:master",
+    "arranger-dashboard": "quay.io/cdis/arranger-dashboard:master",
```

Comment:
remove arranger, we don't use it anymore

Reply:
removed